suplemon/config.py

```diff
@@ -20,25 +20,28 @@ def __init__(self, app):
         self.config_filename = "suplemon-config.json"
         self.keymap_filename = "suplemon-keymap.json"
         self.home_dir = os.path.expanduser("~")
-        self.fpath = os.path.join(self.home_dir, ".config", "suplemon")
+        self.config_dir = os.path.join(self.home_dir, ".config", "suplemon")
 
         self.defaults = {}
         self.keymap = {}
         self.config = {}
 
+    def get_config_dir(self):
+        return self.config_dir
```

Comment:
I like having getters for stuff that I feel I might need more controll over in the future. There's no real reason at the moment, but I guess there might be in the future. This was mainly a "just in case" decision.